streams/src/main/java/org/apache/kafka/streams/KafkaClientSupplier.java

```diff
@@ -26,7 +26,7 @@
 /**
  * {@code KafkaClientSupplier} can be used to provide custom Kafka clients to a {@link KafkaStreams} instance.
  *
- * @see KafkaStreams#KafkaStreams(org.apache.kafka.streams.Topology, StreamsConfig, KafkaClientSupplier)
+ * @see KafkaStreams#KafkaStreams(org.apache.kafka.streams.Topology, java.util.Properties, KafkaClientSupplier)
```

Comment:
nit: `java.util.` can be removed (note, we only specify the whole package for `Topology`, because otherwise we would need to add an `import` statement and get a warning about "unused imports" and a failing build.